apm-lambda-extension/extension/apm_server.go

```diff
@@ -123,10 +133,21 @@ func PostToApmServer(client *http.Client, agentData AgentData, config *extension
 	return nil
 }
 
+// IsTransportStatusHealthyOrPending returns whether the APM server transport status is
+// healthy or pending.
+//
+// This function is public for use in tests.
```

Comment:
```suggestion
// IsTransportStatusHealthyOrPending returns true if the APM server transport status is 
// healthy or pending, and false otherwise.
//
// This function is public for use in tests.
```